src/lib/validation/limit-cart-items.js

```diff
@@ -5,7 +5,6 @@ import { logger } from '../logger';
 export const limitCartItems = (input : any)  => {
   const { cartLimit } = constants;
   if (input.items && input.items.length > cartLimit) {
-    logger.error(`Total number of items in cart is greater than ${ cartLimit }, cart will be truncated`);
```

Comment:
If we're going to remove this you can remove the import as well

Reply:
ah thanks for catching, and also good question (referring to your earlier comment - i should've clarified it in the PR description)